libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx

```diff
@@ -109,8 +117,17 @@ export const DealTicket = ({
             onSelect={(type) => {
               if (type === OrderType.TYPE_LIMIT) {
                 setValue('timeInForce', OrderTimeInForce.TIME_IN_FORCE_GTC);
```

Comment:
nitpick: these setValue('timeInForce') are not needed anymore since [this](https://github.com/vegaprotocol/frontend-monorepo/pull/1779) has been merged

Reply:
when I merged with master this whole bit got deleted as in the mentioned PR